app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+    for person in people:
+        person_list.append(Person(person.get("name"), person.get("age")))
+    for person in people:
+        if "wife" in person and person.get("wife"):
+            husband = Person.people.get(person.get("name"))
```

Comment:
Уou should access the attribute by key without using a method:
1. What happens if Person.people.**get**(person.get("name")) doesn't get a person? We move to the next line `husband.wife` and get an error that None has no attribute wife
2. If we use person.**get**("name"), and it gives us None, then the same problem as above ☝
So if we don't want the result to be None, there is no need to use get method

Reply:
Fix everywhere (besides conditions, of course)

Reply:
fixed, but wondering what is the best general practice?
to use a .get() or access by [key]

Reply:
Depends on case. If your logic suggests that the value could be None, you should use get
In this case, your code will give unexpected errors like `AttributeError None has no attribute wife` instead of `KeyError <name> is not in person dictionary keys`, something like this